tests/virt_autotest/virt_utils.pm

```diff
@@ -303,10 +303,10 @@ sub generate_guest_asset_name {
 
     #get build number
     my $build_num;
-    #for clone job, get build number from SCC proxy which is set in Media
-    if (get_var('CASEDIR')) {
-        get_var('SCC_URL') =~ /^http.*all-([\d\.]*)\.proxy\.*/;
-        $build_num = $1;
+    # for a clone job, the setting BUILD must be set as it was in its original job, for example, BUILD=98.1
+    # or the job will not know in which build guest assets should be download or uploaded
+    if (get_var('CASEDIR') and get_var('BUILD') !~ /^\d+.?\d*$/) {
```

Comment:
SLE Micro job group uses build number in the form of number_anothernumber. But at the moment, no guest asset functionality being used in SLE Micro.

Reply:
thanks, I'll take '_' into account even if there has not requirement of guest assets reusing for SLEM